src/default-envrc

```diff
@@ -0,0 +1,6 @@
+if type -P lorri &>/dev/null; then
+  eval \"$(lorri direnv)\"
+else
+  echo "consider using lorri: https://github.com/nix-community/lorri"
```

Comment:
A quibble about this text (and apologies for delayed bikeshedding):

I can only think of two ways this would print: if you ran `lorri init` (or otherwise set up your .envrc to use lorri), or if you got an .envrc somehow and didn't know about lorri.

In the first case, you already considered (and decided to use!) lorri, and it's somehow not available.

In the second case, you have no idea what lorri is or in what circumstances you'd use it. Or maybe even where this message is coming from.

```suggestion
  echo "while direnv evaluated .envrc, couldn't find the `lorri' command [https://github.com/nix-community/lorri]"
```